dataduct/steps/executors/custom_check.py

```diff
@@ -0,0 +1,21 @@
+import pandas.io.sql as pdsql
+
+from dataduct.data_access import redshift_connection
+
+
+def get_redshift_data(sql):
+    connection = redshift_connection()
+    data = pdsql.read_sql(sql, connection)
+    connection.close()
+    return data.iloc[0][0]
+
+
+def validate_redshift_result(sql, min_result, max_result):
+    result = get_redshift_data(sql)
+
+    if min_result and result < min_result:
+        raise Exception("result {0} is less than min {1}".format(result, min_result))
+    if max_result and result > max_result:
+        raise Exception("result {0} is greater than max {1}".format(result, max_result))
+
+    print result
```

Comment:
would it be useful for this method to return the result value at all?

Reply:
I don't know how we would use a return value here. This function is called as a sort of callback from the custom step script in dataplatform. All I know how to do is throw an exception, or not throw one 😄

Reply:
oh yah, my comment doesn't make much sense now that I think about it.